client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -297,9 +298,36 @@ const Editor = (props: EditorProps): JSX.Element => {
     // TODO this should *probably* be set on focus
     editorRef.current = editor;
     data.editor = editor;
+
+    const storedAccessibilityModes = () => {
+      const accessibilityMode = {
+        isAccessibilityModeOn: props.inAccessibilityMode
+      };
+
+      type AccessibilityMode = typeof accessibilityMode;
+
+      const accessibility = store.get('accessibilityMode') as AccessibilityMode;
+      if (!accessibility) {
+        store.set('accessibilityMode', accessibilityMode);
+      }
+      // Only able to set the arialabel when it is set to true
+      // Otherwise it gets overwritten by the monaco default aria-label
+      if (accessibility?.isAccessibilityModeOn) {
+        editor.updateOptions({
+          ariaLabel:
+            'Accessibility mode set to true press ctrl + e to disable; Press alt F1 for more options'
+        });
+      }
+
+      return accessibility?.isAccessibilityModeOn ?? false;
+    };
+
+    const accessibilityMode = storedAccessibilityModes();
     editor.updateOptions({
-      accessibilitySupport: props.inAccessibilityMode ? 'on' : 'auto'
+      accessibilitySupport: accessibilityMode ? 'on' : 'off'
```

Comment:
Do we need to change from a default of `auto`?  It seems to be fine and is recommended by the Monaco docs: https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.ieditoroptions.html#accessibilitysupport
```suggestion
      accessibilitySupport: accessibilityMode ? 'on' : 'auto'
```

Reply:
'auto' is effectively 'off' for me.  I'm not sure exactly why that's the default, but that's what they suggest in their docs :man_shrugging: